pkg/cmd/browse/browse.go

```diff
@@ -87,6 +87,7 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 			"help:arguments": heredoc.Doc(`
 				A browser location can be specified using arguments in the following format:
 				- by number for issue or pull request, e.g. "123"; or
+				- by commit SHA for commit page, e.g. "77507cd94ccafcf568f8560cfecde965fcfa63": or
```

Comment:
Since commit SHAs are large and unwieldy, I think an example here just adds noise and that we would be better without it. 
```suggestion
				- by commit SHA;
```

Reply:
Thank you for review.
Indeed, the example is noisy.
I had removed it.